Firestore/tests/System/AggregateQueryTest.php

```diff
@@ -186,23 +278,47 @@ public function getWhereCases()
             ['value' => ['foo', 'bar']],
             ['value' => ['foo']]
         ];
+        $numDoc = [
+            ['value' => [1, 2]],
+            ['value' => [30]]
+        ];
+        $maxValueExceedDoc = [
+            ['value' => PHP_FLOAT_MAX],
+            ['value' => PHP_FLOAT_MAX]
+        ];
         $randomVal = base64_encode(random_bytes(10));
+        $randomInt = random_int(1, PHP_INT_MAX);
         return [
             // For testing where: equality for random value
             ['count', null, '=', $randomVal, 1, [['value' => $randomVal]]],
+            ['sum', 'value', '=', $randomInt, $randomInt, [['value' => $randomInt]]],
```

Comment:
Should this value here not be `2*$randomInt`? 
Reason: In both cases, `['value' => $randomVal]` is added. So, when the query filters for docs with `'value' = $randomValue`, it should see 2 docs instead of one!

Reply:
Can you explain this? I see the test case pass though.. but I am not sure what's happening. 
I do not see any place where the docs added in previous test case are filtered out in query OR deleted before the next test case runs?

Reply:
We have used `setUp` instead of `setUp` before class in all the system tests of Firestore.

* **setUp**: This method is always run before running each test, hence it will reinitialise the resource if they are initialised in `setUp`.
* **setUpBeforeClass**: This method is run just once before running any of the tests of the class.